t/test_bhutan.rb

```diff
@@ -0,0 +1,43 @@
+
+require 'csv_to_popolo'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.